articles/quickstart/webapp/aspnet-core-3/_includes/_login.md

```diff
@@ -328,13 +243,55 @@ When the user selects the **Log In** button, the OIDC middleware redirects them
 8. The OIDC middleware extracts the user information from the claims on the ID Token.
 9. The OIDC middleware returns a successful authentication response and a cookie which indicates that the user is authenticated. The cookie contains claims with the user's information. The cookie is stored, so that the cookie middleware will automatically authenticate the user on any future requests. The OIDC middleware receives no more requests, unless it is explicitly challenged.
 
-### Store the Tokens
+## Obtain an Access Token for Calling an API
+
+If you want to call an API from your MVC application, you need to obtain an Access Token issued for the API you want to call. To obtain the token, pass an additional `audience` parameter containing the API identifier to the Auth0 authorization endpoint, which you can get from the API's [Application Settings](${manage_url}/#/applications).
+
+In the configuration for the `OpenIdConnectOptions` object, handle the `OnRedirectToIdentityProvider` event and add the `audience` parameter to `ProtocolMessage`.
+
+```csharp
+// Startup.cs
+
+public void ConfigureServices(IServiceCollection services)
+{
+    // Some code omitted for brevity...
+
+    services.AddAuthentication(options => {
+        //...
+    })
+    .AddCookie()
+    .AddOpenIdConnect("Auth0", options => {
+        // ...
+
+        options.Events = new OpenIdConnectEvents
+        {
+            OnRedirectToIdentityProvider = context =>
+            {
+                context.ProtocolMessage.SetParameter("audience", Configuration["Auth0:Audience"]);
```

Comment:
Can we have a code comment here about why this line is needed? It's the crux of the snippet.